tools/clang/test/CodeGenSPIRV/spv.intrinsicStorageClass.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %dxc -T ps_6_0 -E main -spirv -Vd
+
+//CHECK: {{%\w+}} = OpVariable {{%\w+}} RayPayloadNV
+//CHECK: {{%\w+}} = OpVariable {{%\w+}} CrossWorkgroup
```

Comment:
Could you please check the type of variable as well?
We want to make sure that `RayPayloadNV` is used for `payload` but it is not used for `foo` and vice versa. I am just worried in the future some developers can update the code and break your implementation.